Source/WebKit/Shared/WebPreferencesDefaultValues.h

```diff
@@ -46,6 +46,12 @@
 #define Webgpuhdr_feature_status Preview
 #endif
 
+#if defined(ENABLE_SUPPORT_HDR_DISPLAY_BY_DEFAULT) && ENABLE_SUPPORT_HDR_DISPLAY_BY_DEFAULT && defined(HAVE_SUPPORT_HDR_DISPLAY) && HAVE_SUPPORT_HDR_DISPLAY
+#define Support_hdr_display_feature_status Stable
```

Comment:
You may need to name this `Supporthdrdisplay_feature_status` to make this work, I forget off hand

Reply:
Changing the name did not fix the problem. And I have to explicitly enable the feature for DRT and WTR.